src/include/migraphx/config.hpp

```diff
@@ -23,11 +23,19 @@
  */
 #ifndef MIGRAPHX_GUARD_CONFIG_HPP
 #define MIGRAPHX_GUARD_CONFIG_HPP
-
+#include <migraphx/version.h>
 namespace migraphx {
 
 #if !defined(MIGRAPHX_USE_CLANG_TIDY) && !defined(DOXYGEN)
-#define MIGRAPHX_INLINE_NS version_1
+
+#define MIGRAPHX_CONCAT_1(x, y) x##y
+#define MIGRAPHX_CONCAT(x, y) MIGRAPHX_CONCAT_1(x, y)
+
+#define MIGRAPHX_VERSION_M MIGRAPHX_CONCAT(MIGRAPHX_VERSION_MAJOR, MIGRAPHX_VERSION_MINOR)
+
+#define MIGRAPHX_VERSION MIGRAPHX_CONCAT(MIGRAPHX_VERSION_M, MIGRAPHX_VERSION_PATCH)
+
+#define MIGRAPHX_INLINE_NS MIGRAPHX_CONCAT(VERSION_, MIGRAPHX_VERSION)
```

Comment:
When `BUILD_DEV` is set we should just set it to `version_1`.

Reply:
Done.